src/components/Widgets.tsx

```diff
@@ -85,22 +84,7 @@ export class FileUploadInput extends React.Component<{
     super(props);
   }
   render() {
-    const input = <input className="file-upload-input" type="file" onChange={this.props.onChange} multiple/>;
-    if (this.props.label) {
-      return <div>
-          <div className="row">
-            <div className="column">
-              <div className="text-input-box-label">{this.props.label}</div>
-            </div>
-            <div className="column">
-              <div>{input}</div>
-            </div>
-          </div>
-          {this.props.error && <div className="text-input-box-error">{this.props.error}</div>}
-        </div>;
-    } else {
-      return input;
-    }
+    return <input id="file-upload-input" type="file" onChange={this.props.onChange} multiple hidden/>;
```

Comment:
Here you can do a `ref` to get a handle to the input element. Then add a function called `open()` that files the click event on it, which is called from a parent component.